rts/System/Threading/SpringThreading.h

```diff
@@ -14,14 +14,12 @@
 
 #if   defined(_WIN32)
 	#include "System/Platform/Win/CriticalSection.h"
-#elif defined(__APPLE__) || !defined(USE_FUTEX)
-	#include "System/Platform/Mac/Signal.h"
-#elif !defined(__APPLE__) && defined(USE_FUTEX)
+#elif defined(USE_FUTEX)
 	#include "System/Platform/Linux/Futex.h"
 #endif
 
 
-#if defined(__MINGW32__) && !defined(_GLIBCXX_HAS_GTHREADS)
+#if   defined(__MINGW32__) && !defined(_GLIBCXX_HAS_GTHREADS)
```

Comment:
Whitespace change but no apple nearby, might want to check if something was missed around here (and remove the ws change)